docs/source/en/conceptual/evaluation.md

```diff
@@ -87,7 +87,7 @@ import torch
 seed = 0
 generator = torch.manual_seed(seed)
 
-images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator, output_type="numpy").images
+images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator).images
```

Comment:
Do we need to specify the `output_type` @sayakpaul or is it ok to just leave it as the default `pil` image?

```suggestion
images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator, output_type="np").images
```

Reply:
Here, I thought we show images, so `pil`.